database/seeds/PostStatusesTableSeeder.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class PostStatusesTableSeeder extends Seeder {
+	/**
+	 * Seed the application's database.
+	 *
+	 * @return void
+	 */
+	public function run() {
+
+		\Illuminate\Support\Facades\DB::table( 'post_statuses' )->insert( [
+			'status' => 'published'
```

Comment:
Strive to use constants instead of strings. Put them to  `PostStatus` model
`const STATUS_PUBLISHED = 'published'`, etc.